### PR TITLE
Added "Create Windows VM" page

### PR DESCRIPTION
**Pull request by nunix**
This page details how to create a Windows VM on Harvester.
The example uses the Windows template for a more generic and standardized approach

### CODE REVIEW

**Review by jtravee — CHANGES_REQUESTED** [review]

File: docs/vm/create-vm.md [hunk 1]
```diff
@@ -15,6 +15,10 @@ Description: Create one or more virtual machines from the Virtual Machines page.
 
 Create one or more virtual machines from the **Virtual Machines** page.
 
+!!! note
+	For creating Windows virtual machines, please refer to [this page](./create-windows-vm.md)
```
Comment (btat) [inline docs/vm/create-vm.md:19]:
```suggestion
	For creating Windows virtual machines, please refer to [this page](./create-windows-vm.md).
```

File: docs/vm/create-windows-vm.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+
+---
+
+# Create a Windows VM
+
+Create one or more virtual machines from the **Virtual Machines** page.
+
+!!! note
+	For creating Linux virtual machines, please refer to [this page](./create-vm.md)
```
Comment (btat) [inline docs/vm/create-windows-vm.md:21]:
```suggestion
	For creating Linux virtual machines, please refer to [this page](./create-vm.md).
```

File: docs/vm/create-windows-vm.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
+---
+sidebar_position: 2
+keywords:
+  - Harvester
+  - harvester
+  - Rancher
+  - rancher
+  - Windows
+  - windows
+  - Create a Windows VM
```
Comment (btat) [inline docs/vm/create-windows-vm.md:10]: Should an entry where VM is expanded out to virtual machine be in the list too?

**Review by btat — CHANGES_REQUESTED** [review]

**Review by nunix — COMMENTED** [review]
Applied suggestions for rephrasing

File: docs/vm/create-windows-vm.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+## Header
+
+* In the `header` section, you have the following options:
+
+    1. Create a single or multiple VM instances.
+    1. Set the VM name.
+    1. [Optional] Provide a description for the VM.
+    1. [Optional] Select the VM template `windows-iso-image-base-template`. This template will add a volume with the `virtio` drivers for Windows.
+
+## Tab: Basics
```
[the remark this answers is not in the document]
Comment (nunix) [reply docs/vm/create-windows-vm.md]: Applied suggestion

File: docs/vm/create-windows-vm.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+* In the `Basics` tab, you have the following options:
+
+    1. Configure the number of `CPU` assigned to the VM.
```
[the remark this answers is not in the document]
Comment (nunix) [reply docs/vm/create-windows-vm.md]: Rephrased to `... number of CPU cores ...`

File: docs/vm/create-windows-vm.md [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+        1. `Name`:  The value `default` is set by default. You can keep it or change it.
+        2. `Network`: The value `management Network` is set by default. You can't change this option if there's no other network created. See [Harvester Network](/networking/harvester-network/) for the full description on how to create new networks.
+        3. `Model`: The value `e1000` is set by default. You can keep it or change it to the other available options.
+        4. `Type`: The value `masquerade` is set by default. You can keep it or change it to the other available option, `bridge`.
+    2. You can add additional using the button `Add Network`.
+
+    ![create-windows-vm-networks](assets/create-windows-vm-networks.png)
+
+!!! warning
+	Changing the `Node Scheduling` settings can impact Harvester features, such as the `Live migration`.
```
[the remark this answers is not in the document]
Comment (nunix) [reply docs/vm/create-windows-vm.md]: Rephrased the to `... impact Harvester features, such as disabling the Live migration`.
The impact is also written in the options of Harvester, but it makes total sense to add it here too. Thank you :)

File: docs/vm/create-windows-vm.md (hunk 3 above)
Comment (nunix) [reply docs/vm/create-windows-vm.md:12]: Add the keyword as suggested. And added it too to `Create VM` as I took it as template, and it wasn't there.

File: docs/vm/create-windows-vm.md [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+
+## Header Section
+
+
+    1. Create a single VM instance or multiple VM instances.
+    1. Set the VM name.
+    1. [Optional] Provide a description for the VM.
+    1. [Optional] Select the VM template `windows-iso-image-base-template`. This template will add a volume with the `virtio` drivers for Windows.
+
+## Basics tab
```
Comment (jtravee) [inline docs/vm/create-windows-vm.md:33]:
```suggestion
## Basics Tab
```

**Review by jtravee — CHANGES_REQUESTED** [review]

**Review by btat — APPROVED** [review]

**Review by jtravee — APPROVED** [review]

**Review by guangbochen — APPROVED** [review]

**Pull request merged**